cime_config/usermods_dirs/NEON/FATES/KONA/shell_commands

```diff
@@ -0,0 +1,6 @@
+./xmlchange NEONSITE=KONA
+./xmlchange PTS_LON=263.38956
+./xmlchange PTS_LAT=39.10828
```

Comment:
How about adding something like
./xmlchange BUILD_COMPLETE=FALSE

Reply:
Unfortunately while this works great for a cloned case, since shell_commands is usually called before the case.build command (?) we can still build and submit the case...

@ekluzek perhaps in this instance we should just remove the two crop user mods on the FATES side? We can add a README to the FATES directory to explain why these two are missing, and then when we add FATES to `./run_neon` we can easily add an error there for running those two sites until they are ready.

Reply:
@adrifoster I think that your proposed solution makes sense.

Reply:
@adrifoster that makes sense to me as well. Thanks for trying the solution of having it error out -- it was worth doing. It's good to know how it functions in actual cases.

Reply:
We couldn't come up with a good way to handle this. So I'm resolving this conversation.